src/stylesheets/components/_toasts.scss

```diff
@@ -0,0 +1,66 @@
+.sds-toast {
+    @include grid-row;
+    @include u-text('semibold'); 
+    @include u-margin-top('2');
+
+    &.sds-toast--info {
+        .sds-toast__content {
+            border: solid 2px;
+            @include u-border('secondary-light');
+        }
+
+        .sds-toast__content__icon {
+            @include u-text('secondary-light')
+        }
+    }
+
+    &.sds-toast--success {
+        .sds-toast__content {
+            border: solid 2px;
+            @include u-border('primary-dark');      
+        }
+
+        .sds-toast__content__icon {
+            @include u-text('primary-dark')
+        }
+    }
+
+    .sds-toast__content {
+        @include u-margin-x('3');
```

Comment:
Replace '3' with just 3 - Also replace all instances throughout this file.

Sam-Styles is forgiving about the variable type, but when you import to angular it throws an error. If it's an integer it should not be encased as string and vice versa.

e.g. these are valid

('2px')
(2) 

These are not valid:

(2px)
('2')